BLL/TicketBL.cs

```diff
@@ -52,7 +57,49 @@ public async Task<Ticket> GetTicketDetails(int? id)
 
         }
 
-        
-        
+        public CreateTicketViewModel GetCreateTicketViewModel(int projId)
+        {
+            Project currProject = _projectRepo.Get(projId);
+
+            List<SelectListItem> currUsers = new List<SelectListItem>();
+
+            currProject.AssignedTo.ToList().ForEach(t =>
```

Comment:
Project.AssignedTo value is empty here because it is a navigation property and the Project table in the database does not know about it.
We need to make another call to the UserProject Repository to get these values